### PR TITLE
Weather

### PR DESCRIPTION
**Pull request by TinbeM**

### CODE REVIEW

File: src/components/Weather/WeatherContainer.js [hunk 1]
```diff
@@ -23,28 +27,29 @@ class WeatherContainer extends Component {
...
 		if (city && country) {
-			this.setState({
-				temperature: data.main.temp,
-				city: data.name,
-				country: data.sys.country,
-				humidity: data.main.humidity,
-				description: data.weather[0].description,
-			})
+			this.setState(
+				[
```
Comment (biljecki) [inline src/components/Weather/WeatherContainer.js]: no need for this to be an array [ ]

File: src/components/Weather/WeatherContainer.js [hunk 2]
```diff
@@ -9,11 +9,15 @@ const API_KEY = "128998aad0525e2158bccaee2aeea152"
 class WeatherContainer extends Component {
 
 	state = {
-		temperature: undefined,
-		city: undefined,
-		country: undefined,
-		humidity: undefined,
-		description: undefined,
+		weatherData: [
```
Comment (biljecki) [inline src/components/Weather/WeatherContainer.js]: you can remove the whole array and set undefined

File: src/components/Weather/WeatherDisplay.js [hunk 3]
```diff
@@ -6,12 +6,14 @@ class Display extends Component {
 	{
 		return(
 			<div>
-				<p>{this.props.city}  {this.props.country}</p>
-				<p>Temperature: {this.props.temperature}</p>
-				<p>Humidity: {this.props.humidity}</p>
+								{console.log(this.props.weatherData)}
```
Comment (biljecki) [inline src/components/Weather/WeatherDisplay.js]: remove console log

File: src/components/Weather/WeatherDisplay.js [hunk 4]
```diff
@@ -6,12 +6,14 @@ class Display extends Component {
...
+
+				<p>{this.props.weatherData[0].city}  {this.props.weatherData[0].country}</p>
```
Comment (biljecki) [inline src/components/Weather/WeatherDisplay.js]: 1) remove [0] after you remove the array from the container
2) always destruct when you write this.props multiple times 
you can destruct as 

> const { city, temperatur... } = this.props.weatherData;

File: src/components/Weather/WeatherContainer.js [hunk 5]
```diff
@@ -9,11 +9,15 @@ const API_KEY = "128998aad0525e2158bccaee2aeea152"
 class WeatherContainer extends Component {
```
Comment (biljecki) [inline src/components/Weather/WeatherContainer.js]: this is a Component, not a Container

Comment (biljecki) [reply src/components/Weather/WeatherContainer.js]: not important since we are just playing around but make sure you understand the difference

Comment (TinbeM) [reply src/components/Weather/WeatherContainer.js]: i thought it was a container since it contains all the "weather" components. :( feels bad man

**Pull request merged**